test/spec/fruits/apple_spec.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe "Apple" do
```

Comment:
Are these files supposed to be in here?

Reply:
Yes, I've introduced an integration testing between the client and `rspec`, and these files are used as a fixtures to test that integration.